ebcl/tools/root/debootstrap.py

```diff
@@ -92,6 +102,38 @@ def _generate_apt_config(self) -> None:
             check=True
         )
 
+        # Copy host files to allow adding additional package manager config
+        # and overwriting of generated apt config.
+        if self.config.host_files:
+            # Copy host files to target_dir folder
+            logging.info('Copy apt config to target dir...')
+            for d in self._find_apt_host_files():
+                self.config.fh.copy_files([{'source': d}], self.config.target_dir)
+
+    def _find_apt_host_files(self):
```

Comment:
```suggestion
    def _find_apt_host_files(self) -> None:
```
Otherwise mypy ignores the function

Reply:
Fixed.